src/gui/static/src/app/components/pages/send-skycoin/send-skycoin.component.ts

```diff
@@ -43,22 +43,27 @@ export class SendSkycoinComponent implements OnInit {
   }
 
   send() {
+    if (this.button.disabled()) {
```

Comment:
__note:__ that's the main change, if it's disabled we should not process any send until it's ready again.